addon/locales/en/translations.coffee

```diff
@@ -59,7 +59,7 @@ I18nTranslationsEn =
     "rate_button_label": "Sure"
     "yes_button_label": "Yes!"
     "no_button_label": "Not really"
-    "app_rate_prompt_title": "Thank you, We appreciate your kindness. Give GoodCity a rating?"
+    "app_rate_prompt_title": "Rate GoodCity! Thank you for your kind donation..."
```

Comment:
@shreyas098 , any specific reason for two extra `.` ?

Reply:
No Specific reason.Copied the same text as it was mentioned in ticket @abhinav098

Reply:
Removed it. @abhinav098